### PR TITLE
Added lease quota count and rate limit count docs

### PR DESCRIPTION
**Pull request by divyaac**
PR: https://github.com/hashicorp/vault-enterprise/pull/4552/files

### CODE REVIEW

File: website/content/api-docs/system/lease-count-quotas.mdx [hunk 1]
```diff
@@ -49,6 +49,12 @@ millions of leases in an automated way, it is recommended to space out the creat
   concept of roles (such as `/auth/approle/`), this will make the quota restrict login
   requests to that mount that are made with the specified role. The request will fail if
   the auth mount does not have a concept of roles, or `path` is not an auth mount.
+- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
+all child namespaces of the specified path namespace will inherit this quota. The request
+will fail if `inheritable` is set to true, while the `path` has a value that is not a namespace mount,
+or if a value for `role` is specified. All global quotas are inheritable by default. All namespace quotas
```
Comment (VioletHynes) [inline website/content/api-docs/system/lease-count-quotas.mdx:55]: I think I'd suggest rewording this for clarity, especially as we made the decision that root quotas can't be made uninheritable:
"All global quotas are inheritable by default. All namespace quotas are not inheritable by default."
-> 
"Root namespace quotas are inheritable by default and this cannot be changed, thereby making them global in reach. All other namespace quotas are not inheritable by default."

File: website/content/api-docs/system/lease-count-quotas.mdx [hunk 2]
```diff
@@ -49,6 +49,12 @@ millions of leases in an automated way, it is recommended to space out the creat
   concept of roles (such as `/auth/approle/`), this will make the quota restrict login
   requests to that mount that are made with the specified role. The request will fail if
   the auth mount does not have a concept of roles, or `path` is not an auth mount.
+- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
```
Comment (VioletHynes) [inline website/content/api-docs/system/lease-count-quotas.mdx:52]:
```suggestion
- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a non-root namespace,
```

Comment (VioletHynes) [reply website/content/api-docs/system/lease-count-quotas.mdx:52]: Wait, actually, this doesn't make sense. Removing this suggestion.

File: website/content/api-docs/system/lease-count-quotas.mdx (hunk 1 above)
Comment (VioletHynes) [inline website/content/api-docs/system/lease-count-quotas.mdx:55]:
```suggestion
all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to true if the path does not specify a namespace. All global quotas are inheritable by default. All namespace quotas
```

Comment (VioletHynes) [reply website/content/api-docs/system/lease-count-quotas.mdx:55]: Suggested rewording of the "This will fail" section. In particular, `role` can never be applied on a namespace path anyway

**Review by VioletHynes — APPROVED** [review]
Looks great!

File: website/content/api-docs/system/lease-count-quotas.mdx [hunk 3]
```diff
@@ -49,6 +49,11 @@ millions of leases in an automated way, it is recommended to space out the creat
...
+  all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to
```
Comment (schavis) [inline website/content/api-docs/system/lease-count-quotas.mdx:53]:
```suggestion
  all child namespaces of the specified path namespace will inherit the quota. The `inheritable` parameter cannot be set to
```

Style correction: avoid "this" as a pronoun

File: website/content/api-docs/system/lease-count-quotas.mdx [hunk 4]
```diff
@@ -49,6 +49,11 @@ millions of leases in an automated way, it is recommended to space out the creat
...
+  true if the `path` does not specify a namespace. All root namespace quotas are inheritable by default.
+  All other namespace quotas are not inheritable by default.
```
Comment (schavis) [inline website/content/api-docs/system/lease-count-quotas.mdx:55]:
```suggestion
  `true` if the `path` does not specify a namespace. Only the quotas associated
  with the root namespace are inheritable by default.
```

File: website/content/api-docs/system/rate-limit-quotas.mdx [hunk 5]
```diff
@@ -44,6 +44,10 @@ the mount to restrict more specific API paths.
   concept of roles (such as `/auth/approle/`), this will make the quota restrict login
   requests to that mount that are made with the specified role. The request will fail if
   the auth mount does not have a concept of roles, or `path` is not an auth mount.
+- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
+  all child namespaces of the specified path namespace will inherit this quota. `inheritable` cannot be set to
+  true if the `path` does not specify a namespace. All root namespace quotas are inheritable by default.
+  All other namespace quotas are not inheritable by default.
```
Comment (schavis) [inline website/content/api-docs/system/rate-limit-quotas.mdx:50]:
```suggestion
- `inheritable` `(bool: false)` - If set to `true` on a quota where `path` is set to a namespace,
  all child namespaces of the specified path namespace will inherit the quota. The `inheritable`
  parameter cannot be set to `true` if the `path` does not specify a namespace. Only quotas
  associated with the root namespace quotas are inheritable by default.
```

Same suggestions as for the other file

**Review by schavis — CHANGES_REQUESTED** [review]

**Review by schavis — APPROVED** [review]
